index.js

```diff
@@ -1,39 +1,137 @@
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
+  enterState("roomOne");
+  let roomInventory = ["Seven Days"];
+  let humanInventory = [];
+  let humanResponse = await ask(`182 Main St. You are standing on Main Street between 
+    Church and South Winooski. There is a door here. A keypad sits on the handle.
+    On the door is a handwritten sign.\n`);
+
+  while (currentState === "roomOne") {
+    if (humanResponse.toLowerCase() === "read sign") {
+      humanResponse = await ask(`The sign says "Welcome to Burlington Code Academy! Come on up to
+      the third floor. If the door is locked, use the code 12345"\n`);
+    } else if (humanResponse.toLowerCase() === "take sign") {
+      humanResponse = await ask("That would be selfish. How will other students find their way?\n");
+    } else if (humanResponse.toLowerCase() === "open door") {
+      humanResponse = await ask("The door is locked. There is a keypad on the door handle.\n");
+    } else if (humanResponse.toLowerCase() === "key in 12345" || humanResponse.toLowerCase() === "enter code 12345") {
+      console.log(`\nSuccess! The door opens. You enter the foyer and the door shuts behind you.\n\n`);
+      enterState("roomTwo");
+      humanResponse = await ask(`You are in a foyer. Or maybe it's an antechamber. Or a vestibule. Or an entryway. 
+      Or an atrium. Or a narthex. But let's forget all that fancy flatlander vocabulary, and just call it a foyer. 
+      In Vermont, this is pronounced
+      "FO-ee-yurr".
+      A copy of Seven Days lies in a corner.\n`);
+    } else if (humanResponse.toLowerCase().includes("enter code") || humanResponse.toLowerCase().includes("key in")) {
+      humanResponse = await ask("Bzzzzt! The door is still locked.\n");
+    } else if (humanResponse.toLowerCase() === "gargle") {
+      humanResponse = await ask("Sorry, I don't know how to gargle.\n");
+    } else {
+      humanResponse = await ask("Evaluate your surroundings! Refer to previous directions.\n");
+    }
+  }
+  
+  while (currentState === "roomTwo") {
+    if(humanResponse.toLowerCase() === 'take paper' && humanInventory.includes('Seven Days')){
+      humanResponse = await ask('You have already picked up all items in this room\n');
+    }
+    else if(humanResponse.toLowerCase() === 'take seven days' && humanInventory.includes('Seven Days')){
+      humanResponse = await ask('You have already picked up all items in this room\n');
+    }
+    else if(humanResponse.toLowerCase() === 'pick up paper' && humanInventory.includes('Seven Days')){
+      humanResponse = await ask('You have already picked up all items in this room\n');
+    }
+    else if(humanResponse.toLowerCase() === 'pick up seven days' && humanInventory.includes('Seven Days')){
```

Comment:
This line is redundant in context of the previous "else if" statement. You should handle both cases in one else if statement like you did in line 101